src/scss/_mixins.scss

```diff
@@ -17,6 +17,15 @@
 		padding-left: oSpacingByName('s4');
 	}
 
+	.o-comments__signed-in-text {
+		@include oTypographySans($scale: 0);
+	}
+
+	.o-comments__signed-in-text--inner {
```

Comment:
@ker-an here it looks like you are using a `modifier` for a `element` in the namespacing.

`--{something}` is a modifier so I think we should update that to save confusion. I am happy to chat about this if BEM naming is something you are not very familiar with

Reply:
no, that makes complete sense - so should it instead be something like `o-comments__signed-in-text--bold`?